compiler/rustc_session/src/config.rs

```diff
@@ -403,6 +403,14 @@ impl OutputTypes {
             OutputType::Metadata | OutputType::DepInfo => false,
         })
     }
+
+    // Returns `true` if any of the output types require linking.
+    pub fn should_link(&self) -> bool {
+        self.0.keys().any(|k| match *k {
+            OutputType::Exe => true,
+            _ => false,
```

Comment:
Please match exhaustively to ensure this function gets looked at if `OutputType` enum is expanded in the future.

Reply:
Done.